smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -289,6 +290,23 @@ contract BridgeBank is BankStorage,
         unlockFunds(_recipient, tokenAddress, symbol, _amount);
     }
 
+    event LogMigrateRowan(
+        address migrator,
+        uint256 amount,
+        address previousToken,
+        address newToken
+    );
+
+    function migrateERowan(
+        uint256 amount
+    ) public whenNotPaused {
+        address previousToken = controlledBridgeTokens["erowan"];
+        address newToken = controlledBridgeTokens["Rowan"];
```

Comment:
This makes the assumption that the symbol stored in the controlledBridgeTokens mapping is uppercased when it should be lowercased as all tokens are lowercased before stored in a mappinng.

Reply:
I don't think that's true for controlledBridgeTokens.  There's no attempt to lower-case the symbol in useExistingBridgeToken; it takes the symbol exactly as it is by doing ```string memory _symbol = BridgeToken(_contractAddress).symbol();```.